### DISCUSSION TITLE
Markor URI

### DISCUSSION THREAD

**Discussion by jakeprice-me** [gsantner/markor#2084] [Q&A] [answered]
I render my markdown notes as HTML and have a link "Open in Obsidian" which uses the obsidian URI and allows me to easily open the file in Obsidian. It looks like this: `obsidian://open?vault=myvault&file=inbox.md`. 

There's various hints of this being possible with Markor (perhaps using the `intent://` URI or similar), but I've not been able to find anything clear or certain, and haven't been able to figure it out myself.

So, does Markor have a `markor://` URI scheme, or does it support `intent://` and if the latter, what should the URI look like?

Any help would be much appreciated.

**Comment by gsantner** [comment, marked as answer]
@jakepricedev Do you think about opening a file by sending a intent (i.e. by java code)? Or by opening some URL?

No, there is no markor:// URI scheme / no custom scheme

**Comment by ChaosNicro** [comment]
Sorry for necromancing this, but it would be nice to have a way to link to larger documents in a todo.txt-task to collect more details for a task. Todo doesn't render markdown-links but can use the open-link-button. The button however, needs a complete URI to work.

**Comment by gsantner** [comment]
You can use Markdown checklists in similar fashion to todo.txt, but also have links and more.
